mobile/src/main/java/rs/readahead/washington/mobile/data/entity/uwazi/mapper/Mappers.kt

```diff
@@ -1,15 +1,32 @@
 package rs.readahead.washington.mobile.data.entity.uwazi.mapper
 
 import rs.readahead.washington.mobile.data.entity.uwazi.*
+import rs.readahead.washington.mobile.data.entity.uwazi.Value
 import rs.readahead.washington.mobile.domain.entity.uwazi.*
 
 /**
  * Mapper for template response
  */
+fun RelationShipEntitiesResponse.mapToDomainModel() = rows?.map {
+    it.mapToDomainModel()
+} ?: emptyList()
+
 fun TemplateResponse.mapToDomainModel() = rows?.map {
     it.mapToDomainModel()
 } ?: emptyList()
 
+fun UwaziRelationShipRow.mapToDomainModel() = RelationShipRow(
+    version = __v,
+    id = _id,
+    values = values?.map { it.mapToDomainModel() } ?: emptyList(),
+    type = type,
+    name = name,
+    entityViewPage = entityViewPage
+)
+fun Value.mapToDomainModel() = rs.readahead.washington.mobile.domain.entity.uwazi.Value(
+    label = label,
```

Comment:
why we are using the package name with the class to import it?

Reply:
to differenciate the Value class between data and domain